dashboard/app/models/script.rb

```diff
@@ -692,6 +692,8 @@ def self.add_script(options, raw_script_levels)
       script_level
     end
     script_stages.each do |stage|
+      # make sure we have an up to date view
+      stage.reload
       stage.script_levels = script_levels_by_stage[stage.id]
```

Comment:
note [reload](https://apidock.com/rails/ActiveRecord/Persistence/reload) returns `self`, so you can inline it as `stage.reload.script_levels = ...` if you wanted.